### PR TITLE
minelogy initialization added

### PR DESCRIPTION
**Pull request by DaddyWesker**
So, I've added initialization of minelogy using current minecraft game version. It's not the most elegant way as i think (maybe it'll be better to convert minelogy to class and put it to the agent and rob as `self.minelogy`). Also it is necessary to initialize minelogy for every script it will be used. It leads to the problem (see `7_explorer.py` for example), when we need to initialize minelogy twice. Though it is currently works.

### CODE REVIEW

**Comment by Necr0x0Der** [comment]
Indeed, it might be better to rework minelogy into a class

File: examples/4_mine_and_craft.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from tagilmo.utils.mathutils import normAngle, degree2rad
 
 import numpy as np
-from minelogy import get_log_names, initialize_minelogy
+from minelogy import minelogy
```
Comment (Necr0x0Der) [inline examples/4_mine_and_craft.py:10]: class name might be better to capitalize (`Minelogy`)

File: examples/4_mine_and_craft.py [hunk 2]
```diff
@@ -201,11 +201,11 @@ def mineWhileInRange(rob):
     rob.sendCommand('attack 0')
 
 # A higher-level skill for getting sticks
-def getSticks(rob):
+def getSticks(rob, minelogy_instance):
```
Comment (Necr0x0Der) [inline examples/4_mine_and_craft.py:204]: It's a subjective preference, but `minelogy_instance` looks a little bit awkward. I'd prefer something more compact... I don't know - `mlogy`, `mclogy`, `mckb`.

File: examples/4_mine_and_craft.py (hunk 1 above)
Comment (DaddyWesker) [reply examples/4_mine_and_craft.py]: > class name might be better to capitalize (`Minelogy`)

Done

File: examples/4_mine_and_craft.py (hunk 2 above)
Comment (DaddyWesker) [reply examples/4_mine_and_craft.py]: > It's a subjective preference, but `minelogy_instance` looks a little bit awkward. I'd prefer something more compact... I don't know - `mlogy`, `mclogy`, `mckb`.

Done

**Review by CICS-Oleg — APPROVED** [review]

**Review by CICS-Oleg — APPROVED** [review]

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**